### PR TITLE
Add button to community page for creating a new post.

### PR DESCRIPTION
**Pull request by wolf4ood**
Hi this PR add the ability to create a post within the community page.
I've reused the `CreatePostFab` widget by passing the current community.

I'm still learning Dart and Flutter, any feedback is appreciated :)

Thanks

### CODE REVIEW

File: lib/pages/create_post.dart [hunk 1]
```diff
@@ -21,15 +21,17 @@ import 'full_post.dart';
 
 /// Fab that triggers the [CreatePost] modal
 class CreatePostFab extends HookWidget {
-  const CreatePostFab();
+  final CommunityView community;
+  const CreatePostFab({this.community});
```
Comment (shilangyu) [inline lib/pages/create_post.dart:25]: Please add a new line between the field and the constructor

File: lib/pages/create_post.dart [hunk 2]
```diff
@@ -127,6 +129,32 @@ class CreatePostPage extends HookWidget {
       ),
     );
 
+    DropdownMenuItem<int> communityDropDownItem(CommunityView e) {
+      return DropdownMenuItem(
```
Comment (shilangyu) [inline lib/pages/create_post.dart:133]: Check the CI for lint errors, I'm not sure if I even like this rule anymore...

**Review by shilangyu — APPROVED** [review]
Thanks! Just some minor formatting issues. LGTM

**Comment by shilangyu** [comment]
Oh also, next time please declare yourself willing to take an issue in the issue itself so we can assign it to you :)

**Review by krawieck — APPROVED** [review]

**Comment by krawieck** [comment]
thank you for your contribution

**Pull request merged**